build.sbt

```diff
@@ -36,14 +37,17 @@ lazy val `zio-s3` = project
   .settings(dottySettings)
   .settings(
     libraryDependencies ++= Seq(
-      "dev.zio"               %% "zio"                         % zioVersion,
-      "dev.zio"               %% "zio-streams"                 % zioVersion,
-      "dev.zio"               %% "zio-nio"                     % "2.0.2",
-      "dev.zio"               %% "zio-interop-reactivestreams" % "2.0.2",
-      "software.amazon.awssdk" % "s3"                          % awsVersion,
-      "software.amazon.awssdk" % "sts"                         % awsVersion,
-      "dev.zio"               %% "zio-test"                    % zioVersion % Test,
-      "dev.zio"               %% "zio-test-sbt"                % zioVersion % Test
+      "dev.zio"                       %% "zio"                         % zioVersion,
+      "dev.zio"                       %% "zio-streams"                 % zioVersion,
+      "dev.zio"                       %% "zio-nio"                     % "2.0.2",
+      "dev.zio"                       %% "zio-interop-reactivestreams" % "2.0.2",
+      "software.amazon.awssdk"         % "s3"                          % awsVersion,
+      "software.amazon.awssdk"         % "sts"                         % awsVersion,
+      "dev.zio"                       %% "zio-test"                    % zioVersion  % Test,
+      "dev.zio"                       %% "zio-test-sbt"                % zioVersion  % Test,
+      "com.dimafeng"                  %% "testcontainers-scala-minio"  % "0.41.4"    % Test,
```

Comment:
please remove test container

Reply:
Hi Regis,

Actually we find that handy to not have a launch containers by hand before running tests.
Could you elaborate on why you don't like it, please?

Reply:
For multiple reason: slow down the execution, more code to add in unit-test, more librairies dependencies etc...
Prefer to stick to docker-compose, since it works pretty well.
thank you